kep/9-capacity-scheduling/README.md

```diff
@@ -202,7 +202,8 @@ spec:
 ```
 
 The definitions of "min" and "max" have been explained above. Max is infinite
-by default, min is 0 by default. The sum of min in all ElasticQuota must be 
+(math.MaxInt64: 9,223,372,036,854,775,807) by default, min is 0 by default.
```

Comment:
Readers may not be interested in the concrete number, so:

```suggestion
(in particular `math.MaxInt64`) by default, min is 0 by default.
```

Reply:
@Gekko0114 (sorry for late answer) this is exactly what I tried to explain previously - sorry if I expressed myself poorly

Reply:
Thank you so much! I fixed it